pkgs/test/texlive/default.nix

```diff
@@ -1,8 +1,8 @@
-{ lib, runCommand, fetchurl, file, texlive, writeShellScript }:
+{ lib, runCommand, fetchurl, file, texlive, writeShellScript, isLatest ? false }:
 
 {
 
-  tlpdbNix = runCommand "texlive-test-tlpdb-nix" {
+  tlpdbNix = if isLatest then null else runCommand "texlive-test-tlpdb-nix" {
```

Comment:
Once the rest of the PR has stabilized: `isLatest` should be replaced by an appropriate test that checks `latest-diff.nix`.